src/fusor/models.py

```diff
@@ -125,85 +119,77 @@ class TranscriptSegmentElement(BaseStructuralElement):
     def check_exons(cls, values):
-        """Check that at least one of {`exon_start`, `exon_end`} is set.
-        If set, check that the corresponding `element_genomic` field is set.
+        """Check that at least one of {`exonStart`, `exonEnd`} is set.
+        If set, check that the corresponding `elementGenomic` field is set.
         If not set, set corresponding offset to `None`
 
         """
-        msg = "Must give values for either `exon_start`, `exon_end`, or both"
-        exon_start = values.get("exon_start")
-        exon_end = values.get("exon_end")
+        msg = "Must give values for either `exonStart`, `exonEnd`, or both"
+        exon_start = values.get("exonStart")
+        exon_end = values.get("exonEnd")
         if (not exon_start) and (not exon_end):
             raise ValueError(msg)
 
         if exon_start:
-            if not values.get("element_genomic_start"):
-                msg = "Must give `element_genomic_start` if `exon_start` is given"
+            if not values.get("elementGenomicStart"):
+                msg = "Must give `elementGenomicStart` if `exonStart` is given"
                 raise ValueError(msg)
         else:
-            values["exon_start_offset"] = None
+            values["exonStartOffset"] = None
 
         if exon_end:
-            if not values.get("element_genomic_end"):
-                msg = "Must give `element_genomic_end` if `exon_end` is given"
+            if not values.get("elementGenomicEnd"):
+                msg = "Must give `elementGenomicEnd` if `exonEnd` is given"
                 raise ValueError(msg)
         else:
-            values["exon_end_offset"] = None
+            values["exonEndOffset"] = None
         return values
 
-    _get_transcript_val = field_validator("transcript")(return_value)
     model_config = ConfigDict(
         json_schema_extra={
             "example": {
                 "type": "TranscriptSegmentElement",
                 "transcript": "refseq:NM_152263.3",
-                "exon_start": 1,
-                "exon_start_offset": 0,
-                "exon_end": 8,
-                "exon_end_offset": 0,
-                "gene_descriptor": {
-                    "id": "normalize.gene:TPM3",
-                    "type": "GeneDescriptor",
+                "exonStart": 1,
+                "exonStartOffset": 0,
+                "exonEnd": 8,
+                "exonEndOffset": 0,
+                "gene": {
+                    "id": "hgnc:12012",
+                    "type": "Gene",
                     "label": "TPM3",
-                    "gene_id": "hgnc:12012",
                 },
-                "element_genomic_start": {
-                    "id": "fusor.location_descriptor:NC_000001.11",
-                    "type": "LocationDescriptor",
+                "elementGenomicStart": {
+                    "id": "ga4gh:SL.2K1vML0ofuYrYncrzzXUQOISRFJldZrO",
+                    "type": "SequenceLocation",
                     "label": "NC_000001.11",
```

Comment:
make sure to remove this -- this would be an appropriate label (if we want to do labels) for a SequenceReference, but not a SequenceLocation